Exiled.API/Features/Pickups/Pickup.cs

```diff
@@ -56,8 +56,8 @@ internal Pickup(ItemPickupBase pickupBase)
             // prevent prefabs like `InventoryItemLoader.AvailableItems[ItemType.GrenadeHE].PickupDropModel` from adding to pickup list
             if (pickupBase.Info.ItemId is ItemType.None)
                 return;
-
-            BaseToPickup.Add(pickupBase, this);
+            if (!BaseToPickup.ContainsKey(pickupBase))
+                BaseToPickup.Add(pickupBase, this);
```

Comment:
Im pretty sure it is useless, for getting/creating pickups we use `Pickup::Get`, who has that check